sentry_sdk/integrations/aws_lambda.py

```diff
@@ -126,6 +196,10 @@ def sentry_to_json(*args, **kwargs):
 
             lambda_bootstrap.to_json = sentry_to_json
         else:
+            lambda_bootstrap.LambdaRuntimeClient.post_init_error = _wrap_init_error(
```

Comment:
Note this will be executed for Python 3.7 only.

Reply:
Yes, since for python environments 2.7 & 3.6 the runtime already handles these issues. So, I need not handle for those scenarios. In those cases runtime generates FaultException for initialization error.